components/VaultForm/CreateForm.tsx

```diff
@@ -0,0 +1,215 @@
+'use client'
```

Comment:
GitHub is telling me that this file was created in this PR,  but we already had it. Didn't we?  Is this a bug in Github's diff? Or did you change this file somehow?

Reply:
I have moved it to a new folder, where the ExploreVault and createVault page is there. I forgot to raise the PR to delete the old folder. I will do it now.

Reply:
Fix the issue, moved back.